test/Microsoft.Azure.ServiceBus.UnitTests/Microsoft.Azure.ServiceBus.UnitTests.csproj

```diff
@@ -30,24 +30,9 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
-    <PackageReference Include="ApiApprover">
-      <Version>6.0.0</Version>
-    </PackageReference>
-    <PackageReference Include="ApprovalTests">
-      <Version>3.0.13</Version>
-    </PackageReference>
-    <PackageReference Include="ApprovalUtilities">
-      <Version>3.0.13</Version>
-    </PackageReference>
-    <PackageReference Include="Mono.Cecil">
-      <Version>0.9.6.4</Version>
-    </PackageReference>
-    <PackageReference Include="PublicApiGenerator">
-      <Version>6.0.0</Version>
-    </PackageReference>
-    <PackageReference Include="WindowsAzure.ServiceBus">
-      <Version>4.1.2</Version>
-    </PackageReference>
+    <PackageReference Include="ApprovalTests" Version="3.0.13" />
+    <PackageReference Include="PublicApiGenerator" Version="6.*" />
```

Comment:
Shouldnt the version be minimum of 6.1.0?

Reply:
It will take the latest minor. PublicApiGenerator follows SemVer, so I figured it would be ok to lock major only and leave the rest dynamic. This way we don't have to update for every minor/patch.
Any objections?

Reply:
So what happens when you already have say 6.0.0 in your local cache? Will it still try to fetch the latest minor (6.1.0) or use the local cache?

Reply:
It will retrieve 6.1.0 and store that in your cache.

Reply:
That means, on every 'build', it will try to not only look at local cache, it will try to make a network call to find the latest version of that nuget?
Rather we should just use `[6.1.0,*)` 
(Not sure what the exact syntax is, but you get the point)

Reply:
`="6.4.0"` (its's the latest). I'll change it to make things simpler if you'd like.